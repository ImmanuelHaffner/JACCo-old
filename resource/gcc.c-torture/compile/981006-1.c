int foo (int a, int b)
{
  __label__ z;
  int x;
  int y;
  static void *p;

  switch (a) {
  case 2:
    x = 4;
    break;
  case 4:
    x = 6;
    break;
  case 8: case 10: case 13: case 11: case 17: case 19:
    x = 7;
    break;
  default:
    x = -1;
    break;
  }
  switch (b) {
  case 2:
    y = 4;
    break;
  case 4:
    y = 6;
    break;
  case 8: case 10: case 13: case 11: case 17: case 19:
    y = 7;
    break;
  default:
    y = -1;
    break;
  }
 z:
  p = &&z;
  return x * y;
}
int main (int argc, char *argv[])
{
  return 1 == foo (argc, argc + 1);
}
