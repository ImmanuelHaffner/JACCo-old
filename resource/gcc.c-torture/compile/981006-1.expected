0
981006-1.c:1:1 keyword int
981006-1.c:1:5 identifier foo
981006-1.c:1:9 punctuator (
981006-1.c:1:10 keyword int
981006-1.c:1:14 identifier a
981006-1.c:1:15 punctuator ,
981006-1.c:1:17 keyword int
981006-1.c:1:21 identifier b
981006-1.c:1:22 punctuator )
981006-1.c:2:1 punctuator {
981006-1.c:3:3 identifier __label__
981006-1.c:3:13 identifier z
981006-1.c:3:14 punctuator ;
981006-1.c:4:3 keyword int
981006-1.c:4:7 identifier x
981006-1.c:4:8 punctuator ;
981006-1.c:5:3 keyword int
981006-1.c:5:7 identifier y
981006-1.c:5:8 punctuator ;
981006-1.c:6:3 keyword static
981006-1.c:6:10 keyword void
981006-1.c:6:15 punctuator *
981006-1.c:6:16 identifier p
981006-1.c:6:17 punctuator ;
981006-1.c:8:3 keyword switch
981006-1.c:8:10 punctuator (
981006-1.c:8:11 identifier a
981006-1.c:8:12 punctuator )
981006-1.c:8:14 punctuator {
981006-1.c:9:3 keyword case
981006-1.c:9:8 constant 2
981006-1.c:9:9 punctuator :
981006-1.c:10:5 identifier x
981006-1.c:10:7 punctuator =
981006-1.c:10:9 constant 4
981006-1.c:10:10 punctuator ;
981006-1.c:11:5 keyword break
981006-1.c:11:10 punctuator ;
981006-1.c:12:3 keyword case
981006-1.c:12:8 constant 4
981006-1.c:12:9 punctuator :
981006-1.c:13:5 identifier x
981006-1.c:13:7 punctuator =
981006-1.c:13:9 constant 6
981006-1.c:13:10 punctuator ;
981006-1.c:14:5 keyword break
981006-1.c:14:10 punctuator ;
981006-1.c:15:3 keyword case
981006-1.c:15:8 constant 8
981006-1.c:15:9 punctuator :
981006-1.c:15:11 keyword case
981006-1.c:15:16 constant 10
981006-1.c:15:18 punctuator :
981006-1.c:15:20 keyword case
981006-1.c:15:25 constant 13
981006-1.c:15:27 punctuator :
981006-1.c:15:29 keyword case
981006-1.c:15:34 constant 11
981006-1.c:15:36 punctuator :
981006-1.c:15:38 keyword case
981006-1.c:15:43 constant 17
981006-1.c:15:45 punctuator :
981006-1.c:15:47 keyword case
981006-1.c:15:52 constant 19
981006-1.c:15:54 punctuator :
981006-1.c:16:5 identifier x
981006-1.c:16:7 punctuator =
981006-1.c:16:9 constant 7
981006-1.c:16:10 punctuator ;
981006-1.c:17:5 keyword break
981006-1.c:17:10 punctuator ;
981006-1.c:18:3 keyword default
981006-1.c:18:10 punctuator :
981006-1.c:19:5 identifier x
981006-1.c:19:7 punctuator =
981006-1.c:19:9 punctuator -
981006-1.c:19:10 constant 1
981006-1.c:19:11 punctuator ;
981006-1.c:20:5 keyword break
981006-1.c:20:10 punctuator ;
981006-1.c:21:3 punctuator }
981006-1.c:22:3 keyword switch
981006-1.c:22:10 punctuator (
981006-1.c:22:11 identifier b
981006-1.c:22:12 punctuator )
981006-1.c:22:14 punctuator {
981006-1.c:23:3 keyword case
981006-1.c:23:8 constant 2
981006-1.c:23:9 punctuator :
981006-1.c:24:5 identifier y
981006-1.c:24:7 punctuator =
981006-1.c:24:9 constant 4
981006-1.c:24:10 punctuator ;
981006-1.c:25:5 keyword break
981006-1.c:25:10 punctuator ;
981006-1.c:26:3 keyword case
981006-1.c:26:8 constant 4
981006-1.c:26:9 punctuator :
981006-1.c:27:5 identifier y
981006-1.c:27:7 punctuator =
981006-1.c:27:9 constant 6
981006-1.c:27:10 punctuator ;
981006-1.c:28:5 keyword break
981006-1.c:28:10 punctuator ;
981006-1.c:29:3 keyword case
981006-1.c:29:8 constant 8
981006-1.c:29:9 punctuator :
981006-1.c:29:11 keyword case
981006-1.c:29:16 constant 10
981006-1.c:29:18 punctuator :
981006-1.c:29:20 keyword case
981006-1.c:29:25 constant 13
981006-1.c:29:27 punctuator :
981006-1.c:29:29 keyword case
981006-1.c:29:34 constant 11
981006-1.c:29:36 punctuator :
981006-1.c:29:38 keyword case
981006-1.c:29:43 constant 17
981006-1.c:29:45 punctuator :
981006-1.c:29:47 keyword case
981006-1.c:29:52 constant 19
981006-1.c:29:54 punctuator :
981006-1.c:30:5 identifier y
981006-1.c:30:7 punctuator =
981006-1.c:30:9 constant 7
981006-1.c:30:10 punctuator ;
981006-1.c:31:5 keyword break
981006-1.c:31:10 punctuator ;
981006-1.c:32:3 keyword default
981006-1.c:32:10 punctuator :
981006-1.c:33:5 identifier y
981006-1.c:33:7 punctuator =
981006-1.c:33:9 punctuator -
981006-1.c:33:10 constant 1
981006-1.c:33:11 punctuator ;
981006-1.c:34:5 keyword break
981006-1.c:34:10 punctuator ;
981006-1.c:35:3 punctuator }
981006-1.c:36:2 identifier z
981006-1.c:36:3 punctuator :
981006-1.c:37:3 identifier p
981006-1.c:37:5 punctuator =
981006-1.c:37:7 punctuator &&
981006-1.c:37:9 identifier z
981006-1.c:37:10 punctuator ;
981006-1.c:38:3 keyword return
981006-1.c:38:10 identifier x
981006-1.c:38:12 punctuator *
981006-1.c:38:14 identifier y
981006-1.c:38:15 punctuator ;
981006-1.c:39:1 punctuator }
981006-1.c:40:1 keyword int
981006-1.c:40:5 identifier main
981006-1.c:40:10 punctuator (
981006-1.c:40:11 keyword int
981006-1.c:40:15 identifier argc
981006-1.c:40:19 punctuator ,
981006-1.c:40:21 keyword char
981006-1.c:40:26 punctuator *
981006-1.c:40:27 identifier argv
981006-1.c:40:31 punctuator [
981006-1.c:40:32 punctuator ]
981006-1.c:40:33 punctuator )
981006-1.c:41:1 punctuator {
981006-1.c:42:3 keyword return
981006-1.c:42:10 constant 1
981006-1.c:42:12 punctuator ==
981006-1.c:42:15 identifier foo
981006-1.c:42:19 punctuator (
981006-1.c:42:20 identifier argc
981006-1.c:42:24 punctuator ,
981006-1.c:42:26 identifier argc
981006-1.c:42:31 punctuator +
981006-1.c:42:33 constant 1
981006-1.c:42:34 punctuator )
981006-1.c:42:35 punctuator ;
981006-1.c:43:1 punctuator }
